be/src/olap/data_dir.cpp

```diff
@@ -906,8 +906,13 @@ void DataDir::disks_compaction_num_increment(int64_t delta) {
 }
 
 Status DataDir::move_to_trash(const std::string& tablet_path) {
-    Status res = Status::OK();
+    if (0 == config::trash_file_expire_time_sec) {
+        LOG(INFO) << "delete tablet dir " << tablet_path
+                  << " directly due to trash_file_expire_time_sec is 0";
+        return io::global_local_filesystem()->delete_directory(tablet_path);
```

Comment:
after delete the tablet_path (it is root_dir/shard/tablet_id/schema_hash),   need delete path root_dir/shard/tablet_id if it is empty.

see line 950:  RETURN_IF_ERROR(delete_tablet_parent_path_if_empty(tablet_path))